cleansweep/volunteers/views.py

```diff
@@ -33,6 +34,18 @@ def add_volunteer(place):
     return render_template("add_volunteer.html", place=place, form=form)
 
 
+@plugin.place_view("/volunteers/download")
```

Comment:
I suggest making the URL as /volunteers.xls

Considering REST principles, it is not a good idea to have verbs in the URLs. Nouns are better as URLs represent resources.